modules/github.js

```diff
@@ -12,13 +19,86 @@ const echo = require("./echo");
 // -----------------------------------------------------------------------------------------
 
 const github = {
+    // -----------------------------------------------------------------------------------------
+    // #region Properties
+    // -----------------------------------------------------------------------------------------
+
+    _prompt: null,
+    _token: null,
     andcultureOrg: "AndcultureCode",
     apiRepositoriesRouteParam: "repos",
     apiRootUrl: "https://api.github.com",
+    configAuthConfigPath: upath.toUnix(path.join(os.homedir(), ".netrc")), // Path to octokit-auth-netrc configuration
```

Comment:
Just out of curiosity (ran into this while I was working on some of the recent project plugin/alias stuff), do we need to use both `path` and `upath` for joining paths? Seems like `upath` should be a safe replacement of `path` altogether, but maybe there are some cases I'm not aware of

Reply:
honestly, i think its just an oversight that i copied along 🤦